### PR TITLE
added draw method and game_end method as well as winner_id to game model

### PR DESCRIPTION
**Pull request by derekmccool**

### CODE REVIEW

File: spec/models/game_spec.rb [hunk 1]
```diff
@@ -117,5 +117,21 @@
       end
     end 
   end
+
+  describe 'game end' do
+    it 'should set finished to 1 on game end' do
```
Comment (JarfigNewton) [inline spec/models/game_spec.rb]: Should "1" be "true"? (I'm guessing you rewrote the test but forgot to adjust the name!)

**Review by JarfigNewton — CHANGES_REQUESTED** [review]
Everything looks good as far as I can tell. The rspec tests pass. But Travis is failing for some reason. The log shows this error repeatedly: "Retrying download gem from https://rubygems.org/ due to error." I wonder if the site was down when this was submitted. Maybe you can try a bundle install and push another commit to see if it works now.

File: spec/models/game_spec.rb (hunk 1 above)
Comment (lvsamuel) [reply spec/models/game_spec.rb]: Good catch Jared. Looking into it right now

Comment (lvsamuel) [reply spec/models/game_spec.rb]: Good catch Jared. Looking into this right now

**Review by JarfigNewton — APPROVED** [review]
All good now!

**Pull request merged**